WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -76,9 +76,17 @@ class AccountSettingsService {
     func getSettingsAttempt(count: Int = 0, completion: ((Result<AccountSettings, Error>) -> Void)? = nil) {
         self.remote.getSettings(
```

Comment:
Nitpick. Worth removing the unnecessary `self.` access while we're at it?

Reply:
Removed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20268/commits/8cc516e6ec4d822f15bd94d72b8f59f1fffaaf7b